azure-pipelines/release.yml

```diff
@@ -57,6 +57,66 @@ stages:
   condition: succeeded()
   jobs:
     - job: E2E
+      pool:
+        vmImage: 'ubuntu-latest'
+      steps:
+      - task: DownloadBuildArtifacts@0
+        inputs:
+          buildType: 'current'
+          downloadType: 'single'
+          artifactName: '$(VAR_ARIFACT_NAME)'
+          downloadPath: '$(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)'
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '10.17.0'
+        displayName: 'Install Node.js'
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: 3.7
+      - script: |
+          echo -e "\e[32m[$(date -u)] LOG: Clone azure-rest-api-specs"
+          git clone --single-branch -b master https://github.com/Azure/azure-rest-api-specs.git ../azure-rest-api-specs
+          echo -e "\e[32m[$(date -u)] LOG: Clone azure-cli"
+          git clone --single-branch -b azure-core-preview https://github.com/Azure/azure-cli.git ../azure-cli
+          echo -e "\e[32m[$(date -u)] LOG: Clone azure-cli-extensions"
+          git clone --single-branch -b master https://github.com/Azure/azure-cli-extensions.git ../azure-cli-extensions
+        displayName: 'Clone Branch'
+      - script: |
+          echo -e "\e[32m[$(date -u)] LOG: Install and upgrade Pip"
+          pip install -U pip
+          echo -e "\e[32m[$(date -u)] LOG: Install virtual environment"
+          pip install --user virtualenv
+          echo -e "\e[32m[$(date -u)] LOG: Create virtual environment"
+          python -m virtualenv /opt/venv
+          echo -e "\e[32m[$(date -u)] LOG: Active venv"
+          . /opt/venv/bin/activate
+          echo -e "\e[32m[$(date -u)] LOG: Install pytest"
+          pip install pytest
+          echo -e "\e[32m[$(date -u)] LOG: Install azdev"
+          pip install azdev
+          echo -e "\e[32m[$(date -u)] LOG: Install autorest"
+          npm install -g autorest
+          echo -e "\e[32m[$(date -u)] LOG: Go to azure-cli folder"
+          cd ../azure-cli
+          echo -e "\e[32m[$(date -u)] LOG: Azdev setup"
+          azdev setup -r ../azure-cli-extensions
+        displayName: 'Prepare env'
+      - script: |
+          CURRENT_VERSION=$(node -p "require('./package.json').version")
+          echo -e "\e[32m[$(date -u)] LOG: CURRENT AZ VERSION: $CURRENT_VERSION"
+          echo -e "\e[32m[$(date -u)] LOG: Active venv"
+          . /opt/venv/bin/activate
+          echo -e "\e[32m[$(date -u)] LOG: Go to azure-cli folder"
+          cd ../azure-cli
```

Comment:
Here should be azure-cli-extensions folder right?

Reply:
Not understand, sync offline

Reply:
add --discover